### PR TITLE
feat(newSyndesisLogo): cleaning up autogeneratred files/code for logo

### LINKED ISSUE #4353: Update Syndesis logo + title
## This is a...

<pre><code>
[X] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Documentation issue or request
</code></pre>

Please update the Syndesis logo and title to the now official version found at http://design.jboss.org/syndesis/

### PR DESCRIPTION
**Pull request by jameslau**
## Issue:
Autogenerated code from minishift overrides the config.json file after it is compiled.

## Solution:
Removed unnecessary code from various json and yml files.

fixes #4353

### CODE REVIEW

File: app/ui/src/app/app.component.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export class AppComponent implements OnInit, AfterViewInit {
   /**
    * Logo and Icon with dark background
    */
-  logoIconDarkBg = 'assets/images/syndesis_logo_icon_darkbkg.png';
+  logoDarkBg = 'assets/images/syndesis_logo_icon_darkbkg.png';
```
Comment (kahboom) [inline app/ui/src/app/app.component.ts]: Can we not use the SVG as we are doing on the community site?

File: install/generator/03-syndesis-ui.yml.mustache [hunk 2]
```diff
@@ -127,21 +127,15 @@
         },
 {{#Productized}}
         "branding": {
-          "logoWhiteBg": "",
-          "logoDarkBg": "",
-          "iconWhiteBg": "assets/images/FuseOnlineLogo_Black.svg",
```
Comment (gashcrumb) [inline install/generator/03-syndesis-ui.yml.mustache]: Actually we want to keep this entry, this is for the product build.

File: install/generator/03-syndesis-ui.yml.mustache [hunk 3]
```diff
@@ -127,21 +127,15 @@
...
-          "iconDarkBg": "assets/images/FuseOnlineLogo_White.svg",
```
Comment (gashcrumb) [inline install/generator/03-syndesis-ui.yml.mustache]: And I think this line too.  This is the bit that is relevant when you set `productBuild` there to `true`.

And yes, I know it's convoluted, I'm honestly not sure why :-)

File: install/generator/03-syndesis-ui.yml.mustache [hunk 4]
```diff
@@ -127,21 +127,15 @@
...
+          "logoDarkBg": "assets/images/syndesis_logo_icon_darkbkg.png",
```
Comment (gashcrumb) [inline install/generator/03-syndesis-ui.yml.mustache]: This should be set to the product image logo, not the upstream one.  This particular file is special as it's the install template.

File: app/ui/src/app/app.component.ts (hunk 1 above)
Comment (jameslau) [reply app/ui/src/app/app.component.ts]: `svg` element has been inserted, replacing `img` element.

screen shot 2019-01-31 at 3 11 03 pm

File: install/generator/03-syndesis-ui.yml.mustache (hunk 3 above)
Comment (jameslau) [reply install/generator/03-syndesis-ui.yml.mustache]: Repeating the same set here:

```
"logoWhiteBg": "",
"logoDarkBg": "",
"iconWhiteBg": "assets/images/FuseOnlineLogo_Black.svg",
"logoDarkBg": "assets/images/syndesis_logo_icon_darkbkg.png",
```

File: install/generator/03-syndesis-ui.yml.mustache (hunk 4 above)
Comment (jameslau) [reply install/generator/03-syndesis-ui.yml.mustache]: Removed `"logoDarkBg": "assets/images/syndesis_logo_icon_darkbkg.png",`

**Comment by jameslau** [comment]
@seanforyou23, the favicon files has been replaced.

**Comment by jameslau** [comment]
@kahboom, the `img` has been replaced with an inline `svg` element.

**Pull request merged**